core/config/project_settings.h

```diff
@@ -108,10 +107,14 @@ class ProjectSettings : public Object {
 	HashMap<StringName, LocalVector<Pair<StringName, StringName>>> feature_overrides;
 
 	LocalVector<String> hidden_prefixes;
-	HashMap<StringName, AutoloadInfo> autoloads;
 	HashMap<StringName, String> global_groups;
 	HashMap<StringName, HashSet<StringName>> scene_groups_cache;
 
+	// UIDs can't be used until ProjectSettings is setup.
```

Comment:
```suggestion
	// UIDs can't be used until ProjectSettings is set up.
```
Minor nitpick, "setup" is a noun